src/muya/lib/config/index.js

```diff
@@ -234,3 +234,4 @@ export const MUYA_DEFAULT_OPTION = {
 // }
 
 export const isInElectron = window && window.process && window.process.type === 'renderer'
+export const isMacOs = window && window.navigator && /Mac/.test(window.navigator.platform)
```

Comment:
We use `isOSX` in the main process configuration file. Do you want to change this but didn't really care about it.